pyproject.toml

```diff
@@ -20,7 +20,7 @@ dependencies = [
     "discretize>=0.10.0",
     "empymod>=2.0.0",
     "geoana>=0.5.0",
-    "geoh5py>=0.9.1,<0.11",
+    "geoh5py>=0.10.0-alpha.1, <0.11",
```

Comment:
stating that simpeg requires at geoh5py 0.10
I am not sure here: does this branch of simpeg works with geoh5 0.9, @domfournier ?
In any case, I suppose it is safer to be restrictive.
For now, geoh5py 0.10-alpha is not published to the public. Since simpeg is supposed to be installed in a conda environment, that should be OK, as long as the environment has a geoh5py 0.10-alpha version (installed from the giithub repo, or from a published pre-release on our private conda or pip repo).
Does that sound right?
@domfournier , @SophieCurinier ?

Reply:
It might still work for 0.9.0, but probably safer to restrict as 0.10 breaks backward compatibility on the value_maps.